pypeit/reduce.py

```diff
@@ -1362,6 +1362,18 @@ def apply_relative_scale(self, scaleImg):
         # Check that scaleimg is set to the correct shape
         if self.scaleimg.size == 1:
             self.scaleimg = np.ones_like(self.sciImg.image)
+        debug = False
```

Comment:
Make debug an argument to apply_relative_scale?

Reply:
Debug is not needed here - it was just a check to make sure the right operation is being applied, and it is - deleted.